src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area "
+                + Math.PI * calculateArea() + " sq. units, radius: "
```

Comment:
You are calculating the area twice here, which is unnecessary. You should call the `calculateArea()` method once and store the result in a variable, then use that variable in your output statement.